packages/web-components/src/components/tree-view/tree-view.mdx

```diff
@@ -0,0 +1,54 @@
+import { ArgTypes, Canvas, Markdown, Meta } from '@storybook/blocks';
+import { cdnJs, cdnCss } from '../../globals/internal/storybook-cdn';
+import * as TreeViewStories from './tree-view.stories';
+
+<Meta of={TreeViewStories} />
+
+# Tree View
+
+[Source code](https://github.com/carbon-design-system/carbon/tree/main/packages/web-components/src/components/tree-view)
+&nbsp;|&nbsp;
+[Usage guidelines](https://www.carbondesignsystem.com/components/tree-view/usage)
+&nbsp;|&nbsp;
+[Accessibility](https://www.carbondesignsystem.com/components/tree-view/accessibility)
+
+## Table of Contents
+
+- [TreeView](#TreeView)
+  - [Overview](#overview)
+  - [With Controlled Expansion](#with-controlled-expansion)
+  - [With Custom Icons](#with-custom-icons)
+  - [Temp](#temp)
+- [Component API](#component-api)
+- [Feedback](#feedback)
```

Comment:
```suggestion
- [Overview](#overview)
- [With Controlled Expansion](#with-controlled-expansion)
- [With Custom Icons](#with-custom-icons)
- [Component API](#component-api)
- [CDN](#cdn)
- [Feedback](#feedback)
```

I've been removing the top-level link in updated docs since it just links back to itself. It's inconsistent in React story docs as well, we'll need to fix those too.